services/app-api/src/postgres/contractAndRates/updateDraftContract.test.ts

```diff
@@ -40,10 +41,185 @@ describe('updateDraftContract', () => {
         )
     })
 
-    it.todo(
-        'updates linked document tables as expected (via upsert, not create)'
-    )
-    it.todo('updates linked contact table as expected (via upsert, not create)')
+    it('updates linked documents as expected in multiple requests', async () => {
+        const client = await sharedTestPrismaClient()
+
+        const draftRateForm1: ContractFormEditable = {
```

Comment:
```suggestion
        const draftContractForm1: ContractFormEditable = {
```

It might be confusing in the future with this naming.